### PR TITLE
allow configuration of etcd host, discovery.etcd.io host

### PR DESCRIPTION
**Pull request by tschuy**
Allow the configuration of the discovery hostname and etcd host via command line arguments or env vars.

(See updated README).

### CODE REVIEW

**Comment by tschuy** [comment]
Fixes #30, #37 
Supersedes #38, #27

**Comment by tschuy** [comment]
Don't merge this just yet!

File: discovery.go [hunk 1]
```diff
@@ -1,23 +1,39 @@
...
+	handling "github.com/coreos/discovery.etcd.io/http"
 )
 
-var addr = flag.String("addr", "", "web service address")
+func init() {
+	viper.SetEnvPrefix("disc")
+	viper.AutomaticEnv()
+
+	pflag.StringP("etcd", "e", "http://127.0.0.1:2379", "etcd endpoint location")
+	pflag.StringP("host", "h", "https://discovery.etcd.io", "base location for computed token URI")
```
Comment (brianredbeard) [inline discovery.go]: instead of "base location" i would say "hostname"

**Comment by brianredbeard** [comment]
Jackpot.  LGTM.
do a little dance

**Pull request merged**